webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatus.stories.js

```diff
@@ -1,32 +1,32 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import PowerStatusInner from './index';
-import Story from '../../../../../../../stories/components/Story';
+import PowerStatus from './index';
```

Comment:
Maybe use the not connected version of the component as this one is failing to load because of the missing data field and missing Redux?
`Uncaught Invariant Violation: could not find react-redux context value; please ensure the component is wrapped in `
and add a small explanation about the use of the connected component instead?